chat_shell/chat_shell/tools/builtin/load_skill.py

```
@@ -56,19 +83,21 @@ class LoadSkillTool(BaseTool):
+        "The skill instructions will be added to the system prompt. "
+        "Note: Within the same response, already loaded skills will be confirmed as active."
     )
     args_schema: type[BaseModel] = LoadSkillInput
 
     # Configuration - these are set when creating the tool instance
     user_id: int
     skill_names: list[str]  # Available skill names for this session
     # Skill metadata from ChatRequest (skill_configs)
-    skill_metadata: dict[str, dict] = {}  # skill_name -> {description, prompt, ...}
+    # skill_name -> {description, prompt, displayName, dependencies, ...}
+    skill_metadata: dict[str, dict] = {}
```

Ruff (0.14.14)

[warning] Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)